src/agora/docoding.cpp

```diff
@@ -51,10 +51,18 @@ EventData DoEncode::Launch(size_t tag) {
   size_t symbol_idx_dl = cfg_->GetDLSymbolIdx(frame_id, symbol_id);
   int8_t* input_ptr =
       cfg_->GetInfoBits(raw_data_buffer_, symbol_idx_dl, ue_id, cur_cb_id);
+  int8_t* scramble_buffer_ptr;
+  scramble_buffer_ptr =
+      (int8_t*)std::calloc(cfg_->NumBytesPerCb(), sizeof(int8_t));
```

Comment:
```suggestion
      auto scramble_buffer_ptr = new int8_t[cfg_->NumBytesPerCb()]();
```

Reply:
I think we should prefer new / delete over malloc / free in new submissions.

Reply:
Yes